src/components/common/main/main.container.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Main from './main.component';
+
+const MainContainer = (props) => (
+  <Main {...props} />
+);
+
+export default connect(
+  (state) => { return { moviesList: state.moviesList }; }
```

Comment:
let's use `mapStateToProps` for selecting the data from the store. and export will be `export default connect(mapStateToProps)(Main);` 
in this case you don't need to create: 
`const MainContainer = (props) => (<Main {...props} />);`